ansible/roles/baseline-maintenance/tasks/main.yml

```diff
@@ -0,0 +1,94 @@
+# Note: this script updates only baseline files for candlepin-throughput
+# performance tests.
+
+- name: Checkout Caracalla
+  git:
+    repo: "{{caracalla_git_repo}}"
+    dest: "{{caracalla_checkout}}"
+    force: yes
+    version: "{{caracalla_branch}}"
+
+# We don't use {{caracalla_checkout}}, because we want to use master
+# branch as base branch. {{caracalla_checkout}} can use different branch
+# e.g. developer adds new functionality in feature branch like you
+# right now [jiri] :-P
+- name: Do force checkout of caracalla (PR)
+  git:
+    repo: "{{ git_repo }}"
+    dest: "{{ git_dir }}"
+    force: yes
+    version: master
+
+- set_fact:
+    current_date: "{{ ansible_date_time.date }}_{{ ansible_date_time.hour }}-{{ ansible_date_time.minute }}"
+
+- name: Create new git branch and do checkout of this branch
+  shell: 'git branch -m bot/{{current_date}}'
+  args:
+    chdir: "{{git_dir}}"
+
+- name: Delete old tmp directory for estimation new baseline
+  shell: 'rm -rf /tmp/estimator'
+
+- name: Create new empty directory for estimator
+  shell: 'mkdir /tmp/estimator/'
+
+# TODO: do not do following steps only for 2.3 branch
```

Comment:
We should not reference 2.3 directly here, just "master", because as soon as we decide to branch 2.3 and master, master will be "2.4", thus the reference will be stale.

Reply:
This is IMHO not possible because `baseline_branch` contains `2.x` and never `master`, because `baseline_branch` is overwritten in: `./ansible/roles/candlepin-checkout/tasks/main.yml` (line 20). When more good results for other branches will be collected, then we will be able to update more baseline files of other branches.

And yes, when master is "2.4", then we will have to update our scripts.

Reply:
This is fine for now. I'll create a story to get us around needing to update the scripts. Thanks for the work you've put in so far! :smile: